Remove WPT svg/shapes/animatedPoints-non-animated.html

karlcow opened the pull request:
#### cf79c72808aa0ee771b94b61ccf96c79b745de84
<pre>
Remove WPT svg/shapes/animatedPoints-non-animated.html
<a href="https://bugs.webkit.org/show_bug.cgi?id=295367">https://bugs.webkit.org/show_bug.cgi?id=295367</a>
<a href="https://rdar.apple.com/154903282">rdar://154903282</a>

Reviewed by Tim Nguyen.

This removes the WPT test and expected from the list of our tests. This
needs to be exported back to WPT main repo.
Per discussion on <a href="https://github.com/w3c/svgwg/issues/706#issuecomment-2985152882">https://github.com/w3c/svgwg/issues/706#issuecomment-2985152882</a>
and after reverting the spec <a href="https://github.com/w3c/svgwg/pull/985">https://github.com/w3c/svgwg/pull/985</a>
this specific test is not needed anymore.

* LayoutTests/imported/w3c/web-platform-tests/svg/shapes/animatedPoints-non-animated-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/svg/shapes/animatedPoints-non-animated.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/svg/shapes/w3c-import.log:

Canonical link: <a href="https://commits.webkit.org/297970@main">https://commits.webkit.org/297970@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/e35269e53f8fa358b5d349f069db66786deca5d1

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/110046 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/29705 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/20137 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/116068 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/60294 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/30383 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/38292 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/83663 "Passed tests") | [⏳ 🧪 win-tests](https://ews-build.webkit.org/#/builders/Win-Tests-EWS "Waiting to run tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/112994 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/24226 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/99104 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/64107 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/23595 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/17245 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/59864 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/93599 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/17295 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/118859 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/37086 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/27490 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/92635 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/37458 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/95372 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/92461 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/24252 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/37447 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/15187 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/32968 "Built successfully") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/36980 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/36642 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/39982 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/38351 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

nt1m reviewed (approved):
Maybe make a WPT to test deprecation/removal too if some of the browsers pass this right now

karlcow committed:
Remove WPT svg/shapes/animatedPoints-non-animated.html
https://bugs.webkit.org/show_bug.cgi?id=295367
rdar://154903282

Reviewed by Tim Nguyen.

This removes the WPT test and expected from the list of our tests. This
needs to be exported back to WPT main repo.
Per discussion on https://github.com/w3c/svgwg/issues/706#issuecomment-2985152882
and after reverting the spec https://github.com/w3c/svgwg/pull/985
this specific test is not needed anymore.

* LayoutTests/imported/w3c/web-platform-tests/svg/shapes/animatedPoints-non-animated-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/svg/shapes/animatedPoints-non-animated.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/svg/shapes/w3c-import.log:

Canonical link: https://commits.webkit.org/297970@main
